src/checkbox/checkbox.js

```diff
@@ -163,6 +167,31 @@ class MaterialCheckbox extends MaterialComponent {
   }
 
   /**
+   * Convenience method for changing the text on the checkbox.
+   * Can also get the label with .label and modify it directly.
+   *
+   * @param {string | Element} value The value to set the property to.
+   * @export
+   */
+  set text(value) {
+    if (this.label_) {
+      this.label_.textContent = value;
+    } else {
+      throw new Error('MaterialCheckbox has no label element.');
+    }
+  }
+
+  /**
+   * Return the label element for the checkbox, if any.
+   *
+   * @return {Element?} The label for the checkbox, if any (null if not).
+   * @export
+   */
+  get label() {
+    return this.label_;
```

Comment:
I might get rid of the setter, in that case; folks can just use the getter and modify its inner nodes (i.e., do the `.textContent` themselves). I'm reluctant to to only have text, since people might want to add text formatting within the label via `<b>` or `<span>` or the like.